Source/WebCore/style/values/primitives/StyleZoomNeededToken.h

```diff
@@ -30,5 +30,14 @@ namespace Style {
 // Token passed around to indicate that the evaluation will need zoom passed in the future.
 struct ZoomNeeded { };
 
+struct ZoomFactor {
+    float value;
+
+    constexpr ZoomFactor() : value(1.0f) { }
+    constexpr explicit ZoomFactor(float v) : value(v) { }
+
+    constexpr operator float() const { return value; }
```

Comment:
I think you are better off without this for now. Not that many places need to evaluate the actual zoom value, and this will create the possibility of more overload ambiguity.

Reply:
The line of code. The "operator float".